pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -483,7 +501,6 @@ def _format_checkpoint_name(
         # check and parse user passed keys in the string
         groups = re.findall(r"(\{.*?)[:\}]", filename)
         if len(groups) >= 0:
-            metrics.update({"epoch": epoch, 'step': step})
```

Comment:
you deleted this because the step and epoch are passed to the function as arguments now, correct?
That's essentially the whole refactor if I understand correctly.

Reply:
Correct.

They are in `metrics` already